pom.xml

```diff
@@ -20,41 +20,41 @@
 
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.forgerock.securebanking.uk</groupId>
-    <artifactId>securebanking-openbanking-uk</artifactId>
+    <groupId>com.forgerock.sapi.gateway.uk</groupId>
+    <artifactId>secure-api-gateway-ob-uk-common</artifactId>
     <packaging>pom</packaging>
-    <version>1.1.9-SNAPSHOT</version>
-    <name>Secure Banking access Toolkit library for Open Banking UK specs</name>
-    <url>https://github.com/SecureBankingAccessToolkit/securebanking-openbanking-uk</url>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Secure API Gateway common and shared libraries for Open Banking UK specs</name>
+    <url>https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-common</url>
 
     <parent>
-        <groupId>com.forgerock.securebanking</groupId>
-        <artifactId>securebanking-parent</artifactId>
-        <version>1.0.6-SNAPSHOT</version>
+        <groupId>com.forgerock.sapi.gateway</groupId>
+        <artifactId>secure-api-gateway-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
         <relativePath />
     </parent>
 
     <properties>
         <!-- https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html -->
         <!-- Overwrites the release property (jdk 9+) for maven-compiler-plugin defined in the securebanking-parent -->
-        <java.version>11</java.version>
+        <java.version>14</java.version>
```

Comment:
why are you changing the java version in this PR?

Reply:
Because we decided to use the LTS of JDK few months ago, aren't? I think that the correct java version must be 14 instead of 11 no LTS.

Reply:
Have you tried building this and running functional tests against it @jorgesanchezperez ?

Reply:
but what does it have to do with the issue and PR description?

PRs need to be as tightly scoped as possible, especially these massive ones.

The scope of the issue is to rename things, you shouldn't be including functional changes too.

Reply:
Rollback java.version property.

Reply:
Added an advertisement comment in the pom file to don't modify or delete the java.version set in the project, needs to be compatible always with IG